### PR TITLE
Add cts/command_buffer/compute/sampled_texture_clear test

### PR DESCRIPTION
**Pull request by natashalee**
Add testing for compute pass copied from dawn repo test TextureZeroInitTest::ComputePassSampledTextureClear

### CODE REVIEW

File: src/suites/cts/command_buffer/compute/sampled_texture_clear.spec.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+import { TestGroup } from '../../../../framework/index.js';
+import GLSL from '../../../../tools/glsl.macro.js';
+import { GPUTest } from '../../gpu_test.js';
+
+export const g = new TestGroup(GPUTest);
+
+g.test('compute pass test that sampled texture is cleared', async t => {
+  const texture = t.device.createTexture({
+    size: { width: 1, height: 1, depth: 1 },
```
Comment (kainino0x) [inline src/suites/cts/command_buffer/compute/sampled_texture_clear.spec.ts]: Without too much trouble, I think we could make this more robust by testing a larger texture. You can just do a similar copy from texture into buffer, but with a larger texture, larger dispatch, and larger buffer, then check the whole buffer at the end. I'd make it pretty large, say 256x256.

File: src/suites/cts/command_buffer/compute/sampled_texture_clear.spec.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
+export const description = `
```
Comment (kainino0x) [inline src/suites/cts/command_buffer/compute/sampled_texture_clear.spec.ts]: nit: I think we'll end up having a number of tests for resource initialization. How about making a new directory just for that? e.g. `src/suites/cts/resource_init/`, `sampled_texture.spec.ts` or `compute/sampled_texture.spec.ts` depending on whether you think we'll probably want to split files based on pipeline type in the long run.

**Comment by natashalee** [comment]
Thanks kai, I pushed the updates!

**Comment by kainino0x** [comment]
Oops, sorry I missed it (no notif on pushes to a PR). thank you!

**Pull request merged**